### DISCUSSION TITLE
702

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#882] [Comments]
# 702

최근에 읽었던 전국이라는 춘추+전국시대 배경의 소설책에서 나온 이야기인데 꽤 와닿는게 많았다. 지리소라는 꼽추가 있었다. 그는 흉측하고 장애도 있어 남들이 멸시했으나 바느질과 키질로 곡식을 불려 10명이나 되는 식구를 먹여 살렸다. 나라에서 전쟁이 터지자 멀쩡한 장정들을 징집했는데 지리소는 팔팔하게 다녀도 징집되지 않았다. 나라에서 성을 쌓는다며 부역을 시켜도 그는 제외였다. 기근이 들어 나라에서 구휼미를 풀 때는 불구라는 이유로 일반인보다 많이 받았다. 그는 많은 식구를 건사하고도 벌이가 남아 주변 사람을 챙겼고 일반인보다 장수했다. 남들이 쓸모없다고 여기는 게 오히려 장점이 된 경우로 무용지용(無用之用) 의 다른 예화라고 할 수 있다. 전국 17권 중 (장자에서 나오는 일화와 같다) 쓸모없는 것도 관..

https://jojoldu.tistory.com/702

**Comment by Invidam** [comment]
오히려 좋아..

**Comment by commitonly** [comment]
좋은글잘보고갑니다 향로님 짧은 글이지만 제법 묵직하게 꽂히네요. "시니어가 조직에 없어서 고민인 주니어 개발자에게는 주변에서 이야기하던 좋은 개발문화를 어정쩡한 시니어의 방해없이 마음대로 도입해볼 수 있는 기회가 되기도 한다." 이 부분 공감이 많이 됩니다.

**Comment by mwma91** [comment]
오... 전국의 시대배경이 웹소에서 흔히 보기 힘든 시대라 저도 재밌게 읽었던 기억이 납니다. 그 때는 제가 학생이었어서 별 생각 없이 지나친 부분인데, 주니어 개발자가 된 지금 다시 보니 의미심장하네요 ㅎㅎ

**Comment by al-bur** [comment]
맞는 것 같아요. 무엇인가를 받아들일 때 긍정적인 관점으로 받아들이려고 노력하면 처음에는 안좋아보였지만 거기에서도 배울 게 있고 그렇더고요ㅎㅎ
